ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/RedHat.yml

```diff
@@ -136,6 +136,39 @@
     - ansible_architecture == "s390x"
   tags: default_java
 
+##############################
```

Comment:
Why is this needed? We should already have the expat from the RHEL/CentOS repos installed.
Also if it does need to be done like this we should do it in a separate role.

Reply:
It seems expat is available if your starting rh6 release is new enough. My original base was 6.3, where this error occurred. With 6.10, expat.x86_64 v2.0.1-13.el6_8 is available. I'm fine with removing the expat changes if there is an agreement that the playbooks will only run against the latest of a specific release.

Reply:
We have some RHEL6.9 machines, but I wouldn't expect it generally to be running against anything as old as 6.3 - thanks for the update